src/main/scala/firrtl/AddDescriptionNodes.scala

```diff
@@ -42,6 +49,13 @@ case class AttributeAnnotation(target: Target, description: String) extends Desc
       case Some(seq) => seq.map(n => this.copy(target = n))
     }
   }
+  override private[firrtl] def dedup: Option[(Any, Annotation, ReferenceTarget)] = this match {
+    case a @ AttributeAnnotation(refTarget: ReferenceTarget, _) =>
+      Some(((refTarget.pathlessTarget, description), copy(target = refTarget.pathlessTarget), refTarget))
+    case a @ AttributeAnnotation(pathTarget: ReferenceTarget, _) =>
+      Some(((pathTarget.pathlessTarget, description), copy(target = pathTarget.pathlessTarget), pathTarget))
```

Comment:
```suggestion
    case AttributeAnnotation(refTarget: ReferenceTarget, _) =>
      Some(((refTarget.pathlessTarget, description), copy(target = refTarget.pathlessTarget), refTarget))
```
Same as comment on `DocStringAnnotation`